lib/workers/global/config/parse/env.ts

```diff
@@ -211,11 +212,30 @@ export async function getConfig(
   });
 
   if (env.GITHUB_COM_TOKEN) {
+    const token = env.GITHUB_COM_TOKEN;
     logger.debug(`Converting GITHUB_COM_TOKEN into a global host rule`);
     config.hostRules.push({
       hostType: 'github',
       matchHost: 'github.com',
-      token: env.GITHUB_COM_TOKEN,
+      token,
+    });
+
+    // allow accessing the central bitrise repo via renovate token
```

Comment:
How come Bitrise is the only datasource which requires this type of special treatment?

Reply:
It is currently the only datasource which uses the Github REST Api. `github-tags` and `github-releases` use GraphQL.

Reply:
why do we use the API for raw contents instead of the public raw files endpoint?

Reply:
This is not only for fetching RAW files. 
`bitrise` also fetches folder contents to gather the available versions. 
see for example https://github.com/bitrise-io/bitrise-steplib/tree/master/steps/activate-ssh-key

https://github.com/renovatebot/renovate/blob/f0c20677986914d0497e2366fb6e2fc48e05a5a5/lib/modules/datasource/bitrise/index.ts#L79

I use the contents API as that way not an additional call is necessary to get the default branch. 

https://raw.githubusercontent.com/bitrise-io/bitrise-steplib/master/steps/activate-ssh-key/3.0.2/step.yml.  
vs.  
https://api.github.com/repos/bitrise-io/bitrise-steplib/contents/steps/activate-ssh-key/3.0.2/step.yml

Reply:
you can replace default branch with `HEAD` 🙃

Reply:
TIL, though still the raw endpoint does not work for folder contents.

Reply:
Also not, sure if this raw endpoints exist on GHE setups.